test/helm3/ct.yaml

```diff
@@ -12,6 +12,10 @@ chart-repos:
   - kubefed-charts=https://raw.githubusercontent.com/kubernetes-sigs/kubefed/master/charts
   - jetstack-charts=https://charts.jetstack.io
   - banzaicloud=https://kubernetes-charts.banzaicloud.com
+<<<<<<< HEAD
```

Comment:
Oops

Reply:
dammit @juliangieseke ;)

I'm really not even sure why those are even there.

Reply:
anyway.. thanks for catching it.

fixed.